src/view/components/formikReactSpectrum3/checkbox.jsx

```diff
@@ -18,11 +18,13 @@ import FieldDescriptionAndError from "../fieldDescriptionAndError";
 import "./checkbox.styl";
 
 const Checkbox = ({ name, description, width, ...otherProps }) => {
-  const [{ value }, { error }, { setValue, setTouched }] = useField(name);
+  const [{ value }, { touched, error }, { setValue, setTouched }] = useField(
+    name
+  );
   return (
     <FieldDescriptionAndError
       description={description}
-      error={error}
+      error={touched && error ? error : undefined}
```

Comment:
I can't just do `touched && error` because when `touched` is `false`, it would feed a `false` value into `FieldDescriptionAndError`.